tools/creating-dev-deployments-for-decisions.adoc

```diff
@@ -24,9 +24,7 @@ Clicking the item on this dropdown will take you to a special web application {C
 This web app lets you select which Decision from your project you want to work with, and provides a form much like DMN Runner's. You get a link for the Swagger UI with your Decisions' API information.
 
 .Dev deployment app for a Decision, created from {CANVAS}
-image::tools/image3.png[tools/image3,width=490,height=404]
-
-*↑ Change this screenshot to one with the {CANVAS} logo?*
+image::tools/image3.png[tools/image3,width=490]
```

Comment:
Removed the height to not distort the image, as the proportion might not be the same as the previous one.